db/users/update_password.go

```diff
@@ -60,25 +59,31 @@ func UpdateUser(ctx context.Context, app, addonUUID, username string) error {
 	isPasswordGenerated := false
 	if password == "" {
 		isPasswordGenerated = true
-		password = gopassword.Generate(64)
-		confirmedPassword = password
 	}
 
-	userUpdateParam := scalingo.DatabaseUpdateUserParam{
-		DatabaseID:           addonUUID,
-		Password:             password,
-		PasswordConfirmation: confirmedPassword,
-	}
-	databaseUsers, err := c.DatabaseUpdateUser(ctx, app, addonUUID, username, userUpdateParam)
-	if err != nil {
-		return errors.Wrap(ctx, err, "update password of the given database user")
-	}
+	var databaseUser scalingo.DatabaseUser
+
+	// We have two different API calls here to avoid breaking backwards compatibility of the CLI
+	if !isPasswordGenerated {
+		userUpdateParam := scalingo.DatabaseUpdateUserParam{
+			DatabaseID:           addonUUID,
+			Password:             password,
+			PasswordConfirmation: confirmedPassword,
+		}
+		databaseUser, err = c.DatabaseUpdateUser(ctx, app, addonUUID, username, userUpdateParam)
+		if err != nil {
+			return errors.Wrap(ctx, err, "update password of the given database user")
+		}
+	} else {
```

Comment:
suggestion: I like returning the function as soon as possible. In this case, you could return here and the `else` is useless. There is less indentation which makes the code easier to read IMO
```suggestion
		}
		
		fmt.Printf("User \"%s\" password updated.\n", databaseUser.Name)
		return nil
	}
```

Reply:
Thanks. I actually misread the original flow. Your suggestion is absolutely correct. Thanks

Reply:
> Did you ask product about the CLI backward compatibility?

Yes. My question was answered by benjamin in the comments in the [STORY-642] in notion. 

It should be (remain) possible to set a user-supplied password using the CLI.